<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="hle_code">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="hbt_login">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>200</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string> 로그인</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CODE :</string>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>20</width>
        <height>271</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>621</x>
        <y>160</y>
        <width>20</width>
        <height>271</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>이름 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hle_name">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>420</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="hbt_add">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>310</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>등록</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hle_add_name">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>이름 :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="hrb_admin">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>390</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>관리자</string>
      </property>
<<<<<<< HEAD
      <property name="checked">
       <bool>true</bool>
      </property>
=======
>>>>>>> f194b85 (230206 UI 생성)
     </widget>
     <widget class="QRadioButton" name="hrb_user">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>390</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>학생</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="stw">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>561</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>home</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>56</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>이름 :</string>
        </property>
       </widget>
       <widget class="QLabel" name="sl_h_name">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>30</y>
          <width>56</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>name</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="slcd_h_point">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>30</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>56</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>point :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>30</y>
          <width>56</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>등급 :</string>
        </property>
       </widget>
       <widget class="Line" name="line_11">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>80</y>
          <width>781</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="sl_h_rating">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>30</y>
          <width>56</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>F</string>
        </property>
       </widget>
       <widget class="Line" name="line_12">
        <property name="geometry">
         <rect>
          <x>385</x>
          <y>90</y>
          <width>20</width>
          <height>441</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QTableWidget" name="stw_h_score">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>100</y>
          <width>371</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="stw_h_progress">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>371</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>학습</string>
       </attribute>
       <widget class="QTableWidget" name="stw_contents">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>771</width>
          <height>401</height>
         </rect>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>번호</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>연도</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>내용</string>
         </property>
        </column>
       </widget>
       <widget class="Line" name="line_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>771</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>50</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>연도 선택 </string>
        </property>
       </widget>
       <widget class="QPushButton" name="study_save_btn">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>20</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>학습내용 저장</string>
        </property>
       </widget>
       <widget class="QPushButton" name="load_study_btn">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>60</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>학습내용 불러오기</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>문제</string>
       </attribute>
       <widget class="QTableView" name="stw_test">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>256</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_14">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>20</width>
          <height>511</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_15">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>320</y>
          <width>501</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="sl_test">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>481</width>
          <height>291</height>
         </rect>
        </property>
        <property name="text">
         <string>문제</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="sle_test">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>340</y>
          <width>431</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sbr_test">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>340</y>
          <width>51</width>
          <height>181</height>
         </rect>
        </property>
        <property name="text">
         <string>확인</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>상담</string>
       </attribute>
       <widget class="Line" name="line_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>771</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QListWidget" name="slw_chat">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>771</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sle_chat">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>771</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>QnA</string>
       </attribute>
       <widget class="QLineEdit" name="sle_qa">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>691</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="stw_qa">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>771</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sbt_qa">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>10</y>
          <width>75</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>등록</string>
        </property>
       </widget>
       <widget class="Line" name="line_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>771</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTabWidget" name="atw">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>560</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>801</width>
        <height>560</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_q">
       <attribute name="title">
        <string>문제만들기</string>
       </attribute>
       <widget class="QPushButton" name="abt_finish">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>등록</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abt_del">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>문제 삭제</string>
        </property>
       </widget>
       <widget class="QComboBox" name="acb_num">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>76</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>등록번호</string>
         </property>
        </item>
       </widget>
       <widget class="QTableWidget" name="atw_q">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>781</width>
          <height>491</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>문제</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>답안</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>배점</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>point</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="abt_cancel">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>취소</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abt_add">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>문제 추가</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>10</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>배점 총합:</string>
        </property>
       </widget>
       <widget class="QLabel" name="al_score">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>10</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_u">
       <attribute name="title">
        <string>학생 관리</string>
       </attribute>
       <widget class="QListWidget" name="alw_user">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>256</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>20</width>
          <height>511</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QTreeWidget" name="atw_record">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>491</width>
          <height>511</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>등록번호/ 문제</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>이름/ 답안</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>점수/ 이름</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>채점</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>소요시간</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_c">
       <attribute name="title">
        <string>상담</string>
       </attribute>
       <widget class="QListWidget" name="alw_chat_user">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>256</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>40</y>
          <width>20</width>
          <height>481</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>771</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="ale_chat">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>771</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="alw_chat">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>491</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_qa">
       <attribute name="title">
        <string>QnA</string>
       </attribute>
       <widget class="QLineEdit" name="ale_qa">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>691</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="abt_qa">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>10</y>
          <width>75</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>등록</string>
        </property>
       </widget>
       <widget class="Line" name="line_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>771</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QTableWidget" name="atw_qa">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>771</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
